### PR TITLE
In 1504: Readable Width for Saves and Search

### PR DESCRIPTION
**Pull request by CMasterson**
## Summary
Adds readable width to Saves and mimics it for Search.

## References 
https://getpocket.atlassian.net/browse/IN-1504

## Implementation Details
Saves is built on UIKit and so we are able to add `section.contentInsetsReference = .readableContent` to the content cells layout config.

Search is built using SwiftUI, which does not appear to have a readableContent setting.
Therefore, if the interface width size class is .regular we restrict the list width to a preset amount (currently 500 points)

## Test Steps
Launch and use Saves and Search as normal.
On iPad you may then enable split screen and while changing the window size you will notice the views adapt to their width.
iPhones do not have a landscape layout for Search and Saves, only Reader which already uses Readable Width.

## PR Checklist:
- [ ] Self Review (review, clean up, documentation, run tests)
- [ ] Basic Self QA

## Screenshots
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 46 23
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 46 17
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 46 12
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 46 04
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 45 59
Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-05-31 at 09 45 51

### CODE REVIEW

File: PocketKit/Sources/PocketKit/Article/ReadableViewController.swift [hunk 1]
```diff
@@ -325,6 +325,7 @@ extension ReadableViewController {
             // Zero out the default leading/trailing contentInsets, but preserve the default top/bottom values.
             // This ensures each section will be inset horizontally exactly to the readable content width.
             var contentInsets = section.contentInsets
+            print("== contentInset \(contentInsets)")
```
Comment (cyndichin) [inline PocketKit/Sources/PocketKit/Article/ReadableViewController.swift:328]: remove?

**Review by cyndichin — APPROVED** [review]
one comment on print statement, otherwise code looks good!

File: PocketKit/Sources/PocketKit/Article/ReadableViewController.swift (hunk 1 above)
Comment (CMasterson) [reply PocketKit/Sources/PocketKit/Article/ReadableViewController.swift]: Done!

**Comment by CMasterson** [comment]
Updated the SwiftUI ReadableWidth mimic to 700 points, seems to work well on iPad.

**Pull request merged**